blueocean-docker-dev-image/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.jenkins.blueocean</groupId>
+        <artifactId>blueocean-parent</artifactId>
+        <version>1.0.0-b07-SNAPSHOT</version>
```

Comment:
Also - will this be deployed as a plugin as part of a release? if to - an entry will need to exist in: https://github.com/jenkins-infra/repository-permissions-updater/tree/master/permissions

Ideally this is NOT deployed (as it is only for dev time use) right?